R/lists.R

```diff
@@ -99,5 +99,6 @@ wpp_countries <- function() {
       "country.name"
     ))
   found_countries <- found_countries[!is.na(found_countries)]
```

Comment:
```suggestion
  found_countries[!is.na(found_countries)]
```

Reply:
Thanks for this! I'm not a regular data.table user so I sometimes get tripped up on the copy-by-reference approach